aiogram/api/types/chat_member.py

```diff
@@ -65,3 +65,9 @@ class ChatMember(TelegramObject):
     inline bots"""
     can_add_web_page_previews: Optional[bool] = None
     """Restricted only. True, if the user is allowed to add web page previews to their messages"""
+
+    def is_chat_admin(self) -> bool:
+        return self.status in {"creator", "administrator"}
+
+    def is_chat_member(self) -> bool:
+        return self.status not in {"left", "kicked"}
```

Comment:
how does it differ from `is_member` field?

Reply:
is_member is optional for restricted users only and returned by the api. True, if the user is a member of the chat at the moment of the request unlike is_chat_member which is evaluated in place.